src/helpers/articles.js

```diff
@@ -47,7 +68,14 @@ class ArticleModel {
         {
           association: 'author',
           attributes: ['username']
-        }
+        },
+        {
+          association: 'tags',
+          attributes: ['name'],
+          through: {
```

Comment:
@frostyblok Nice observation 👍 , I added that so that the return value doesn't also return the attributes of the join table, because they'er not needed

Reply:
Wow! I did not know that, thank you.